code/components/voip-server-mumble/src/ban.cpp

```diff
@@ -201,8 +201,9 @@ message_t *Ban_getBanList(void)
 	msg = Msg_banList_create(bancount);
 	list_iterate(itr, &banlist) {
 		ban = list_get_entry(itr, ban_t, node);
-		//gmtime_r(&ban->time, &timespec);
+		gmtime_s(&timespec, &ban->time);
```

Comment:
Does this exist on Linux as well?

Reply:
Good catch, `gmtime_r` is a POSIX thread safe alternative to `gmtime` and `gmtime_s` is for MSVC, thread-safety doesn't matter that much in this context right?